unicoremc/tests/test_views.py

```diff
@@ -82,18 +83,117 @@ def test_create_new_project(self):
         self.assertEqual(
             project.get_frontend_custom_domain_list(),
-            ['http://za.ffl.qa-hub.unicore.io'])
+            ['http://za-ffl.qa-hub.unicore.io'])
         self.assertEqual(
             project.cms_url(),
             'http://cms.ffl-za-%s.qa-hub.unicore.io' % project.id)
         self.assertEqual(
-            project.cms_custom_domain, 'cms.za.ffl.qa-hub.unicore.io')
+            project.cms_custom_domain, 'za-ffl.qa-content.unicore.io')
         self.assertEqual(
             project.get_cms_custom_domain_list(),
-            ['http://cms.za.ffl.qa-hub.unicore.io'])
+            ['http://za-ffl.qa-content.unicore.io'])
+        self.assertEqual(project.external_repos.count(), 1)
+        self.assertTrue(project.own_repo())
+        self.assertEqual(len(project.all_repos()), 2)
+        self.assertEqual(project.organization.slug, 'foo-org')
+
+        workspace = self.mk_workspace(
+            working_dir=settings.CMS_REPO_PATH,
+            name='ffl-za',
+            index_prefix='unicore_cms_ffl_za')
+
+        self.assertEqual(workspace.S(Category).count(), 1)
+        self.assertEqual(workspace.S(Page).count(), 1)
+        self.assertEqual(workspace.S(EGLocalisation).count(), 1)
+
+        cat = Category({
+            'title': 'Some title',
+            'slug': 'some-slug'
+        })
+        workspace.save(cat, 'Saving a Category')
+
+        page = Page({
+            'title': 'Some page title',
+            'slug': 'some-page-slug'
+        })
+        workspace.save(page, 'Saving a Page')
+
+        loc = EGLocalisation({
+            'locale': 'spa_ES',
+            'image': 'some-image-uuid',
+            'image_host': 'http://some.site.com',
+        })
+        workspace.save(loc, 'Saving a Localisation')
+
+        workspace.refresh_index()
+
+        self.assertEqual(workspace.S(Category).count(), 2)
+        self.assertEqual(workspace.S(Page).count(), 2)
+        self.assertEqual(workspace.S(EGLocalisation).count(), 2)
+
+        self.addCleanup(lambda: shutil.rmtree(
+            os.path.join(settings.CMS_REPO_PATH, 'ffl-za')))
+
+    @responses.activate
+    @override_settings(HUB_SUBDOMAIN='qa-apollo')
+    def test_create_new_project_for_apollo(self):
+        existing_project = self.mk_project()
+
+        self.client.login(username='testuser2', password='test')
+        self.client.get(
+            reverse('organizations:select-active', args=('foo-org',)))
+
+        self.mock_create_repo()
+        self.mock_create_webhook()
+        self.mock_create_hub_app()
+        self.mock_create_unicore_distribute_repo()
+        self.mock_create_marathon_app()
+
+        app_type = AppType._for(
+            'ffl', 'Facts for Life', 'springboard',
+            'universalcore/unicore-cms-ffl')
+
+        data = {
+            'app_type': app_type.id,
+            'base_repo': self.base_repo_sm.repo.git_dir,
+            'project_repos[]': existing_project.own_repo().pk,
```

Comment:
Oh, that's a multi select field. It's a forms syntax for fields that can return multiple values